src/rdkafka.c

```diff
@@ -1087,22 +1135,19 @@ static void rd_kafka_stats_emit_all (rd_kafka_t *rk) {
 			   rd_atomic64_get(&rkb->rkb_c.rx_partial),
                            rd_atomic64_get(&rkb->rkb_c.zbuf_grow),
                            rd_atomic64_get(&rkb->rkb_c.buf_grow),
-                           rd_atomic64_get(&rkb->rkb_c.wakeups),
-			   int_latency.ra_v.minv,
-			   int_latency.ra_v.maxv,
-			   int_latency.ra_v.avg,
-			   int_latency.ra_v.sum,
-			   int_latency.ra_v.cnt,
-			   rtt.ra_v.minv,
-			   rtt.ra_v.maxv,
-			   rtt.ra_v.avg,
-			   rtt.ra_v.sum,
-			   rtt.ra_v.cnt,
-			   throttle.ra_v.minv,
-			   throttle.ra_v.maxv,
-			   throttle.ra_v.avg,
-			   throttle.ra_v.sum,
-			   throttle.ra_v.cnt);
+                           rd_atomic64_get(&rkb->rkb_c.wakeups));
+
+                total.tx       += rd_atomic64_get(&rkb->rkb_c.tx);
+                total.tx_bytes += rd_atomic64_get(&rkb->rkb_c.tx_bytes);
+                total.rx       += rd_atomic64_get(&rkb->rkb_c.rx);
+                total.rx_bytes += rd_atomic64_get(&rkb->rkb_c.rx_bytes);
+
+                rd_kafka_stats_emit_avg(st, "int_lantecy",
```

Comment:
what is lantecy?

Reply:
It is a trigger word used to make developers write tests.